app/controllers/preprints/provider/moderation.js

```diff
@@ -26,6 +26,10 @@ export default Controller.extend(Analytics, moderationQueryParams.Mixin, {
     store: service(),
     theme: service(),
 
+    hasPermission: computed('model', function () {
```

Comment:
If this is defined in `moderation.js`, what happens if the user goes directly to settings or the detail page without permissions?